chat-plugins/info.js

```diff
@@ -1337,6 +1337,15 @@ exports.commands = {
 		if (showAll || target === 'tiering' || target === 'tiers' || target === 'tier') {
 			buffer.push("<a href=\"https://www.smogon.com/ingame/battle/tiering-faq\">Tiering FAQ</a>");
 		}
+		if (showAll || toId(target).startsWith('lostpass') || target === 'pwreset') {
```

Comment:
What about just `password`? I don't think the average user would think to try something like `pwreset` without having known about the command